tokio/src/sync/notify.rs

```diff
@@ -105,6 +105,13 @@ pub struct Notify {
     state: AtomicU8,
     waiters: Mutex<WaitList>,
 }
+#[derive(Debug, Clone, Copy)]
+enum NotificationType {
```

Comment:
nit: i'd add a newline here (surprised that rustfmt doesn't do that?)
```suggestion
}

#[derive(Debug, Clone, Copy)]
enum NotificationType {
```